src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -221,20 +224,24 @@ public void visit(NodeTraversal t, Node n, Node parent) {
           Node callNode = node;
           String requiredNamespace = callNode.getLastChild().getString();
           String localName = n.getFirstChild().getQualifiedName();
-          convertRequireToImportStatements(n, localName, requiredNamespace, false);
+          convertRequireToImportStatements(
+              n, Collections.singletonList(localName), requiredNamespace, false);
           return;
         }
 
-        // var {foo} = goog.require(...);
+        // var {foo, bar} = goog.require(...);
         if (node.isObjectPattern()
             && node.getNext().getFirstChild() != null
             && node.getNext().getFirstChild().matchesQualifiedName("goog.require")) {
-          // TODO(#392): Support multiple destructured import values here.
-          //     Currently, this only allows for a single destructured import value.
-          String namedExport = node.getFirstChild().getString();
+          Node importedNode = node.getFirstChild();
+          List<String> namedExports = new ArrayList();
+          while (importedNode != null) {
+            namedExports.add(importedNode.getString());
+            importedNode = importedNode.getNext();
+          }
           String requiredNamespace = node.getNext().getFirstChild().getNext().getString();
-          requiredNamespace += "." + namedExport;
-          convertRequireToImportStatements(n, namedExport, requiredNamespace, true);
+          requiredNamespace += "." + namedExports.get(0);
```

Comment:
TL;DR: pick any element from the namedExports it will work.

Reasons:
In `convertRequireToImportStatements()` function, we use `requiredNamespace` as a key to a few HashMaps for different purposes. For example there's a `namespaceToModule` HashMap that we use it to get the file module.
For non destructuring imports, this key is just the `requiredNamespace`. For multiple destructuring imports, the key is  \`${requiredNamespace}.${namedExport}\`. For example, given ` const {foo, bar} = goog.require('a');`, I would expect the `namespaceToModule` HashMap to contain ('a', fileModule), but instead it contains ('a.foo', fileModule) and ('a.bar', fileModule). They map to the same fileModule.

This variable should be renamed to `requiredNamespaceOrRequiredNamespacePlusNamedExportForDestructuringImports` to reflect what it really is.

Reply:
I see. Thanks for explaining.